dracut/parse-kickstart

```diff
@@ -492,12 +494,13 @@ def ksnet_to_ifcfg(net, filename=None):
 
         for i, slave in enumerate(net.bondslaves.split(","), 1):
             slave_ifcfg = {
+                            'DEVICE': slave,
                             'TYPE' : "Ethernet",
                             'NAME' : "%s slave %s" % (dev, i),
                             'UUID' : str(uuid.uuid4()),
                             'ONBOOT' : "yes" if net.activate else "no",
                             'MASTER' : ifcfg['UUID'],
-                            'HWADDR' : readsysfile("/sys/class/net/%s/address" % slave),
+                            #'HWADDR' : readsysfile("/sys/class/net/%s/address" % slave),
```

Comment:
Isn't a better way to remove it completely instead of commenting it out. It is still in git history.